packages/server/lib/browsers/index.coffee

```diff
@@ -36,6 +37,7 @@ cleanup = ->
   instance = null
 
 getBrowserLauncherByFamily = (family) ->
+  debug("getBrowserLauncherByFamily for %s", family)
```

Comment:
```suggestion
  debug("getBrowserLauncherByFamily %o", { family })
```

Reply:
i find it much easier to read debug logs like this, since the actual variable name is highlighted in the log this way